whisper-fine-tuning-sprint/README.md

```diff
@@ -0,0 +1,617 @@
+# Whisper fine-tuning sprint 🤗
+
+Welcome to the Whisper fine-tuning sprint 🎙️ !
+
+The goal of this event is to build **robust**, **real-world** speech recognition (ASR) systems in as many languages as possible 🌏🌍🌎.
```

Comment:
Maybe we can be a bit more specific here? i.e. relate to Whisper and achieving SoTA 70+ langs?